.github/workflows/performance.yml

```
@@ -9,10 +9,13 @@ on:
     tags:
       - '[6-9]+.[0-9]'
       - '[0-9]+.[0-9].[0-9]+'
+      - '![45].[0-9].[0-9]+'
+      - '!6.[01].[0-9]+'
```

We can probably simplify these to:

```suggestion
      - '[6-9]+.[2-9]'
      - '[6-9]+.[2-9].[0-9]+'
```